.gitignore

```diff
@@ -36,3 +36,4 @@ package-lock.json
 # MARK: Test Files
 
 tests_output
+scribe_data_json_export/*
```

Comment:
Great idea, @SethiShreya :) I'll also remove the current ones from the repo :)

Reply:
Or hmmm, not quite yet as we need the current translations. But soon :)

Reply:
Removed all the data with Wikidata as its source and also the SQLite directory :) Also added all other output types.